src/main/java/bio/terra/cli/command/shared/options/ResourceCreation.java

```diff
@@ -18,7 +18,7 @@
   @CommandLine.Option(
       names = "--cloning",
       description =
-          "Instructions for handling when cloning the workspace: ${COMPLETION-CANDIDATES}")
+          "Instructions for handling when cloning the workspace: ${COMPLETION-CANDIDATES}.")
```

Comment:
also when cloning the resource

Reply:
updated description to "cloning the workspace or resource"